features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/App.js

```diff
@@ -35,10 +37,17 @@ import './App.css'
 class Protected extends Component {
     constructor(props) {
         super(props);
-        this.state = {showLeftMenu: false};
+        this.state = {showLeftMenu: false, authConfigs: null};
         this.setLeftMenu = this.setLeftMenu.bind(this);
         message.config({top: '48px'}); // .custom-header height + some offset
         /* TODO: need to fix the header to avoid conflicting with messages ~tmkb*/
+        this.handleResponse = this.handleResponse.bind(this);
+        /* TODO: Get apim base url from config*/
+        Axios.get("https://localhost:9292/login/login/publisher").then(this.handleResponse);
```

Comment:
Do not hardcode the URL

Reply:
We have already created a git issue for this and will be tracked via https://github.com/wso2/carbon-apimgt/issues/5087.